src/OpenTelemetry/Trace/TracerProviderSdk.cs

```diff
@@ -240,7 +239,7 @@ internal sealed class TracerProviderSdk : TracerProvider
             else if (this.sampler is AlwaysOffSampler)
             {
                 listener.Sample = (ref ActivityCreationOptions<ActivityContext> options) =>
-                    !Sdk.SuppressInstrumentation ? PropagateOrIgnoreData(options.Parent) : ActivitySamplingResult.None;
+                    !Sdk.SuppressInstrumentation ? ActivitySamplingResult.PropagationData : ActivitySamplingResult.None;
```

Comment:
Am I correct that this `None` was only used as a special case for `AlwaysOffSampler`? The `ComputeActivitySamplingResult` already seems to return `ActivitySamplingResult.PropagationData` for `DROP`.